server/src/utils/fight/generateFight.ts

```diff
@@ -147,6 +147,7 @@ const generateFight = async ({
     team2: { brutes: team2.brutes ?? [], backups: team2Backups, bosses: team2.bosses ?? [] },
     modifiers,
     clanFight: clanWar,
+    mapEffect: (background as FightBackground).effect,
```

Comment:
After typing the boss and tournament backgrounds correctly, you don't need to force type here

```suggestion
    mapEffect: background.effect,
```

Reply:
Oh I get it now